ang/afsearchECKEntityTypes.aff.json

```diff
@@ -3,7 +3,7 @@
     "title": "ECK Entity Types",
     "icon": "fa-cubes",
     "server_route": "civicrm/admin/eck/entity-types",
-    "permission": "administer CiviCRM",
+    "permission": "administer eck entity types",
```

Comment:
This will cause a superficial git conflict with #90

Reply:
I rebased the branch after merging and corrected the permission in the Afform definition.

Reply:
Thanks @jensschuppe